core/deployment/src/main/java/io/quarkus/deployment/steps/ConfigurationSetup.java

```diff
@@ -164,57 +155,31 @@
             NameIterator.class);
     private static final MethodDescriptor ARDCS_CTOR = MethodDescriptor.ofConstructor(AbstractRawDefaultConfigSource.class);
 
+    private static final MethodDescriptor CS_POPULATE_CONVERTERS = MethodDescriptor.ofMethod(ConverterSupport.class,
+            "populateConverters", void.class, ConfigBuilder.class);
+
     private static final String CONFIG_ROOTS_LIST = "META-INF/quarkus-config-roots.list";
     private static final String[] NO_STRINGS = new String[0];
 
     public ConfigurationSetup() {
     }
 
-    @BuildStep
-    public void setUpConverters(BuildProducer<ConfigurationCustomConverterBuildItem> configurationTypes) {
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                InetSocketAddress.class,
-                InetSocketAddressConverter.class));
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                CidrAddress.class,
-                CidrAddressConverter.class));
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                InetAddress.class,
-                InetAddressConverter.class));
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                Pattern.class,
-                RegexConverter.class));
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                CipherSuiteSelector.class,
-                CipherSuiteSelectorConverter.class));
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                Protocol.class,
-                ProtocolConverter.class));
-        configurationTypes.produce(new ConfigurationCustomConverterBuildItem(
-                200,
-                Path.class,
-                PathConverter.class));
-    }
-
     /**
      * Run before anything that consumes configuration; sets up the main configuration definition instance.
      *
-     * @param converters the converters to set up
      * @param runTimeConfigConsumer the run time config consumer
      * @param buildTimeConfigConsumer the build time config consumer
      * @param buildTimeRunTimeConfigConsumer the build time/run time fixed config consumer
+     * @param resourceConsumer
+     * @param niResourceConsumer
+     * @param runTimeDefaultConsumer
      * @param extensionClassLoaderBuildItem the extension class loader build item
      * @param archiveRootBuildItem the application archive root
+     * @throws IOException
```

Comment:
Please avoid changes not related to the issue as it makes it hard to review.

Reply:
Sure, but I will leave these javadocs I added here. I noticed that the number of params didn't match arguments count here so I added missing ones. Indeed, Github's code review is not very readable.